src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9saktillos/K9SakTilLosAdapterTjeneste.kt

```diff
@@ -271,31 +285,154 @@ class K9SakTilLosAdapterTjeneste(
         OppgaveFeltverdiDto(
             nøkkel = "totrinnskontroll",
             verdi = event.aksjonspunktTilstander.filter { aksjonspunktTilstandDto ->
-                aksjonspunktTilstandDto.aksjonspunktKode.equals("5015") && aksjonspunktTilstandDto.status !in (listOf(AksjonspunktStatus.AVBRUTT))
+                aksjonspunktTilstandDto.aksjonspunktKode.equals("5015") && aksjonspunktTilstandDto.status !in (listOf(
+                    AksjonspunktStatus.AVBRUTT
```

Comment:
Det er litt skummelt å utlede om det er totrinn på nytt. På behandlingen i k9-sak finnes det et felt som heter "toTrinnsBehandling" som jeg tror vi kan benytte (men det blir jo da kun for nye eventer).

Reply:
Da må vi legge på mer på eventene som sendes fra k9-sak og evt ha helgardering her i koden - bruke flagg fra behandling når det ligger på eventet, og bruke denne koden ellers? Lurer på om jeg har lyst til å legge det i future work-skuffen inntil videre?